core/rebuild_async_stubs.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env deno run --allow-read --allow-write
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
```

Comment:
Would be nice to add some tests for this template system

Reply:
Also if you'd prefer to do it in Rust, I'd recommend `tinytemplate` for nicer templating :)